packages/strapi-admin/admin/src/components/Roles/Permissions/requiredActions.js

```diff
@@ -0,0 +1,5 @@
+import { staticAttributeActions } from './utils';
+
+const requiredActions = staticAttributeActions;
+
+export default requiredActions;
```

Comment:
It seems unnecessary but no. It is a CE file. The EE file equivalent to this one is different